find_green_build

```diff
@@ -39,9 +40,13 @@ PROG=${0##*/}
 #+ OPTIONS
 #+     [--official]          - Produce an official release version for a
 #+                             release branch
-#+     --github-token=       - Must be specified if GITHUB_TOKEN not set
-#+     --exclude-suites=     - Space separated list of CI suites to exclude
+#+     [--github-token=]     - Must be specified if GITHUB_TOKEN not set
+#+     [--exclude-suites=]   - Space separated list of CI suites to exclude
 #+                             from go/nogo criteria
+#+     [--allow-stale-build] - Keep going even if the build is behind HEAD,
+#+                             just for informational purposes.
+#+     [--show-all-suites]   - Show the status of all suites for each build,
```

Comment:
Whenever I see a new flag, I pause to consider if this is adding more options/complexity where we don't need to and I think this is one of those times.  Let's just make that the default.